client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,352 @@
+					{ translate(
+						'Registering a .FR domain'
+					) }
+				</h1>
+				<p className="registrant-extra-info__form-desciption">
+					{ translate(
+						'Almost done! We need some extra details to register domains ending in ".fr".'
+					) }
+				</p>
+				<FormFieldset>
+					<FormLegend>
+						{ translate( "Who's this domain for?" ) }
+					</FormLegend>
+					<FormLabel>
+						<FormRadio value="individual"
+							id="registrantType"
+							checked={ 'individual' === registrantType }
+							onChange={ this.handleChangeEvent } />
+						<span>{ translate( 'An individual' ) }</span>
+					</FormLabel>
+
+					<FormLabel>
+						<FormRadio value="organization"
+							id="registrantType"
+							checked={ 'organization' === registrantType }
+							onChange={ this.handleChangeEvent } />
+						<span>{ translate( 'A company or organization' ) }</span>
+					</FormLabel>
+				</FormFieldset>
+
+				{ 'individual' === registrantType
+					? this.renderPersonalFields()
+					: this.renderOrganizationFields() }
+
+				{ this.props.children }
+			</form>
+		);
+	}
+
+	renderPersonalFields() {
+		const translate = this.props.translate;
+		const screenReaderText = 'screen-reader-text';
+		const {
+			countryOfBirth,
+			dobDays,
+			dobMonths,
+			dobYears,
+			placeOfBirth,
+			postalCodeOfBirth,
+		} = { ...emptyValues, ...this.props.values };
+
+		return (
+			<div>
+				<FormFieldset>
+					<FormLabel htmlFor="countryOfBirth">
+						{ translate( 'Country of Birth' ) }
+					</FormLabel>
+					<FormCountrySelect
+						id="countryOfBirth"
+						value={ countryOfBirth }
+						countriesList={ this.props.countriesList }
+						className="registrant-extra-info__form-country-select"
+						onChange={ this.handleChangeEvent } />
+				</FormFieldset>
+
+				<FormFieldset>
+					<FormLegend>
+						{ translate( 'Date of Birth' ) }
+					</FormLegend>
+					<div className="registrant-extra-info__dob-inputs">
+						<div className="registrant-extra-info__dob-column">
+							<FormLabel htmlFor="dobYears" className={ screenReaderText }>
+								{ translate( 'Year' ) }
+							</FormLabel>
+							<FormTextInput className="registrant-extra-info__dob-year"
+								id="dobYears"
+								value={ dobYears }
+								type="number"
+								placeholder={ translate( 'YYYY', {
+									comment: 'Placeholder text for the year part of a date input. Indicates that the user should enter a year as a 4 digit value', // eslint-disable-line max-len
+								} ) }
+								onChange={ this.handleDobChangeEvent } />
+						</div>
+						<div className="registrant-extra-info__dob-column">
+							<FormLabel htmlFor="dobMonths" className={ screenReaderText }>
+								{ translate( 'Month' ) }
+							</FormLabel>
+							<FormTextInput className="registrant-extra-info__dob-month"
+								id="dobMonths"
+								value={ dobMonths }
+								max="2"
+								type="number"
+								placeholder={ translate( 'MM', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'mm', { context: 'millimeters'} )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).